packages/ui-components/src/__tests__/components/Chart/parseChartConfig.test.js

```diff
@@ -4,8 +4,11 @@
  *
  */
 
-import { parseChartConfig } from '../parseChartConfig';
-import { CHART_COLOR_PALETTE, EXPANDED_CHART_COLOR_PALETTE } from '../constants';
+import { parseChartConfig } from '../../../components/Chart/parseChartConfig';
```

Comment:
Update ui-components folder structure to put all tests under `__tests__` instead of alongside each component for consistency with all other packages.